.travis.yml

```diff
@@ -1,7 +1,7 @@
 sudo: false
 language: node_js
 node_js:
-  - "4.1"
+  - "6.1"
```

Comment:
Makes sense, I took the example from `widgets`. I'll do the same with the package template PR